src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -406,6 +410,25 @@ def __repr__(self):
             f"implementation = {self.getImplementationType()}"
         )
 
+    def verifyValidFieldName(func_name: str, field_name: str):
+        if func_name and func_name not in [
+            "DENORMALIZE_PERMX",
+            "NORMALIZE_PORO",
+            "LN",
+            "LOG10",
+            "EXP0",
+            "LOG",
+            "EXP",
+            "TRUNC_POW10",
+            "LN0",
+            "DENORMALIZE_PORO",
+            "NORMALIZE_PERMZ",
+            "NORMALIZE_PERMX",
+            "DENORMALIZE_PERMZ",
+            "POW10",
+        ]:
+            raise ValueError(f"FIELD {field_name}:{func_name} is an invalid function")
```

Comment:
looks great! only thing i would like to ask about is if it's at all possible to get the list of functions from where they're defined in the C code - or would that be too much trouble to become a bit DRYer?

Reply:
No this is definitely something i already considered doing.
I guess i also should update the docs while i'm at it. The list of available functions wasn't fully complete either.
I'll give it a shot!